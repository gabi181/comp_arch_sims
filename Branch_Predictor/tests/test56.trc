32 7 4 0 local_history local_tables not_using_share
0x0E13A31C N 0x57897094
0x0E13A31C N 0x57897094
0x37A4C7C8 T 0x3353012C
0x4352607C N 0x0379A5E4
0x71E41EB8 N 0x269249A0
0x7B3F0A0C T 0x2E685B5C
0x0E13A31C T 0x57897094
0x28AF0B3C T 0x6137EBA8
0x71E41EB8 T 0x269249A0
0x4352607C T 0x0379A5E4
0x0E13A31C N 0x57897094
0x37A4C7C8 N 0x3353012C
0x37A4C7C8 N 0x3353012C
0x71E41EB8 N 0x269249A0
0x28AF0B3C N 0x6137EBA8
0x0E13A31C T 0x57897094
0x4352607C N 0x0379A5E4
0x37A4C7C8 T 0x3353012C
0x0E13A31C N 0x57897094
0x7B3F0A0C T 0x2E685B5C
0x7F75BE3C N 0x198EC9B8
