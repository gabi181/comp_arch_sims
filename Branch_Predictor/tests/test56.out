0xe13a31c N 0xe13a320
0xe13a31c N 0xe13a320
0x37a4c7c8 N 0x37a4c7cc
0x4352607c N 0x43526080
0x71e41eb8 N 0x71e41ebc
0x7b3f0a0c N 0x7b3f0a10
0xe13a31c N 0xe13a320
0x28af0b3c N 0x28af0b40
0x71e41eb8 N 0x71e41ebc
0x4352607c N 0x43526080
0xe13a31c N 0xe13a320
0x37a4c7c8 N 0x37a4c7cc
0x37a4c7c8 N 0x37a4c7cc
0x71e41eb8 N 0x71e41ebc
0x28af0b3c N 0x28af0b40
0xe13a31c N 0xe13a320
0x4352607c N 0x43526080
0x37a4c7c8 N 0x37a4c7cc
0xe13a31c N 0xe13a320
0x7b3f0a0c N 0x7b3f0a10
0x7f75be3c N 0x7f75be40
flush_num: 9, br_num: 21, size: 9536b
